lib/seattleflu/id3c/cli/command/etl/redcap_det_childcare.py

```diff
@@ -28,8 +28,7 @@ def __init__(self, project_id: int, lang: str, command_name: str) -> None:
         self.command_name = command_name
 
 PROJECTS = [
-        ChildcareProject(23740, 'en', 'childcare'),
-        ChildcareProject(29351, 'en', 'childcare-2021')
+        ChildcareProject(29351, 'en', 'childcare')
```

Comment:
Should we leave in the old project just in case there are data updates in the future? We can just remove the automated cronjob but still leave a way for us to run the ETL manually for the old project.

Reply:
Ok. If it's handy to still have this in the ETL, I'll just go ahead and drop that last commit.